WordPress/Classes/Utility/ReachabilityUtils.h

```diff
@@ -6,14 +6,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (BOOL)isInternetReachable;
 
-+ (void)showAlertNoInternetConnection;
```

Comment:
I wanted to convert the whole `ReachabilityUtils` to Swift but some tests fail. Perhaps later!

Reply:
which tests? maybe its worth documenting here so we can create a future task for it?

Reply:
I should have noted those. I'll convert this to Swift again and add a comment.